packages/nimble-components/src/table/specs/editable-table-ixd.md

```diff
@@ -0,0 +1,103 @@
+# Nimble Editable Table (IxD)
+
+## Overview
+
+Nimble table support for interactive cell editing.
+
+### Background
+
+-   [Figma worksheet](https://www.figma.com/design/r2yGNQNVFdE7cBO9CyHmQx/Nimble---IxD?node-id=1221-36463)
+-   [Input table columns (select, text field, etc) #1190](https://github.com/ni/nimble/issues/1190)
+
+## Usage
+
+**When to use:**
+
+-   When users need to quickly and efficiently update data directly within the table.
+-   When inline editing can improve the user experience by reducing the need for separate forms or dialogs.
+-   When the table is used in scenarios where data accuracy can be ensured through validation and error handling mechanisms.
+
+**When not to use:**
+
+-   When data integrity is critical and the table should not be modified by users.
+-   When the table is used for displaying static or read-only data.
+-   When the user interface requires a more complex data entry form.
+
+## Anatomy
+
+![Editable table anatomy](./spec-images/editable-cell-anatomy.png)
+
+| Element               | Description                                                              |
+| --------------------- | ------------------------------------------------------------------------ |
+| Static cell value     | Non-editable display of cell value                                       |
+| Action button         | Button to open cell context menu. Appears on row hover or keyboard focus |
+| Editable cell control | Text or numeric control that supports entering or editing cell data      |
+
+## Behavior
+
+### States
+
+![Editable cell states](./spec-images/editable-cell-states.png)
+
+#### Error State
+
+![Editable cell error states](./spec-images/editable-cell-error-data.png)
+
+### ARIA Considerations
+
+-   Maintain compatibility with existing keyboard navigation behavior
+-   Consider resolving [existing ARIA gaps](https://github.com/ni/nimble/issues/2285)
+
+### Mouse Interactions
```

Comment:
I'd strongly prefer that we maintain the edit state even if the cell scrolls out of view, such that it can scroll back into view in the same state. While it's true the menu button dropdown closes on scroll, setting or reverting an edited cell value is a much more aggressive action in response to something as simple as scrolling.

Do we have any options for persisting this type of state as the cell is scrolled out of view?

Reply:
If that proves too expensive, we should prevent the table from scrolling while a cell is in edit focus.

Reply:
Discussed this a bit with Fred today. Maintaining an edited cell as 'still being edited' after a scroll will probably be difficult due to virtualization - as soon as you scroll more than 1 row's height, even if the row is still visible in the table viewport, the row elements have been re-bound to different data at that point. We can remember which row/cell was being edited (and the current edited cell value), but we'd need to reapply that to a new row visual again (since now a different row visual represents the data row that was being edited, after the scroll), and there might be visual artifacts/ visible flashing if we do that.

Assuming we can prevent mousewheel scrolling the table while editing a cell, that's probably the simplest approach (least work to implement).